### DISCUSSION TITLE
How to change timezone

### DISCUSSION THREAD

**Discussion by KevMyst** [Budibase/budibase#4054] [Help]
When add new row on budibase i see January 12 2022, 00:00. On SQL Server datetime column = 2022-01-11 23:00:00.000

**Comment by mjashanks** [comment]
Hi @KevMyst - we are aware of this issue, and have chatted about it in here: https://github.com/Budibase/budibase/issues/1504

We will address this soon. Thanks for reporting.

**Comment by joebudi** [comment]
Hi @KevMyst  
I'm the co-founder of Budibase. Would you be interested in jumping on a quick call to discuss how you are using Budibase, and any feedback you may have? This would mean the world to us, and help us build a better product 👍 .
https://calendly.com/budibase/joe
Cheers,
Joe
